docs/pages/blog/nextgen-pickers.md

```diff
@@ -0,0 +1,130 @@
+
+## The new date and time fields
+
+This new input type we're talking about is a set of components shortly named [Fields](https://next.mui.com/x/react-date-pickers/fields/).
+They are the base for the new pickers and a revolution in terms of keyboard usability.
+
+In contrast with the masked inputs that only display simple text behind a formatting mask, and were often cumbersome, especially when editing dates, the new fields are smart and specialized in editing date and time numbers.
+It can recognize what's a month, day, or whatever piece of the date you are currently editing and act accordingly.
+
+For example, watch what happens when you edit the month with arrow keys.
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/nextgen-pickers/intelligent-field.mp4" type="video/mp4" />
+</video>
+
+As you increase the month, the component automatically watches for valid values throughout each piece of your date input, like you're navigating in a calendar.
+
+The Fields are a perfect match to the calendar view, but they are available as stand-alone components, and you can import them from the latest v6 package.
+
+```jsx
+import { Unstable_DateField as DateField } from '@mui/x-date-pickers/DateField';
+
+<DateField label="My first field" />;
+```
+
+They are also included by default on the new pickers.
+
+```jsx
+import { Unstable_NextDatePicker as NextDatePicker } from '@mui/x-date-pickers/NextDatePicker';
+
+<NextDatePicker label="My first v6 picker" />;
+```
+
+## Enhanced keyboard usability
+
+With the new fields, not only can you type the date as a text, but you can use your arrow keys to quickly navigate through each piece of date and time values.
+You can edit them atomically, without, for instance, leaking the year into the month, a common occurrence with masked inputs.
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/nextgen-pickers/keyboard-nav.mp4" type="video/mp4" />
+</video>
+
+For a quick comparison, let's check out, for instance, how the two different approaches behave on the most typical scenarios.
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/nextgen-pickers/fields-common-usecases.mp4" type="video/mp4" />
+</video>
+
+Try out the live demo of the `DateField` below.
+
+<iframe
+  src="https://codesandbox.io/embed/date-field-demo-pb87v0?fontsize=12&hidenavigation=1&module=%2F"
+  style="width:100%; height:200px; border:0; border-radius: 4px; overflow:hidden;"
+  allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+  sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+></iframe>
```

Comment:
If we want to keep this codesandbox, I think that we should freeze the versions. I would expect it to break before the end of the year:

<img width="454" alt="Screenshot 2023-01-01 at 23 35 41" src="https://user-images.githubusercontent.com/3165635/210186458-3fd8130a-0a46-433d-96ad-95b7d475f4ac.png">

We could also make the codesandbox part of the MUI org so anybody in the team can make edits without changing the sandbox. Steps:

1. open https://www.notion.so/mui-org/Accounts-945f0ccb9f7749ccb9e84b9eb9b13ec7#04a515471469433daa20fb1a44b1525f and follow the link to join the org.
2. fork 

<img width="291" alt="Screenshot 2023-01-01 at 23 38 07" src="https://user-images.githubusercontent.com/3165635/210186519-06597044-223a-4905-88b8-3a717fe9ca93.png">

Reply:
I froze the versions on the sandbox, but I can't fork it with MUI.
(The pickers still need to get updated when beta is actually out)

<img width="294" alt="image" src="https://user-images.githubusercontent.com/550141/213293886-cead5b9b-afbf-42cb-bc3d-9a7bb670bff4.png">

Reply:
> I froze the versions on the sandbox

Great, it was the most important

> I can't fork it with MUI

We are supposed to be under an Open Source plan which gives us unlimited editors. They got something wrong, I'm letting them know.